website/docs/docs/dbt-versions/release-notes/03-Oct-2023/dbt-deps-auto-install.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Enhancement: dbt Cloud auto-installs 'dbt deps' on startup"
+description: "October 2023 :The dbt Cloud IDE and dbt Cloud CLI auto-handles 'dbt deps' on startup; manual run needed for 'packages.yml' changes. Available for multi-tenant users (single-tenant support coming soon) and applies to all dbt versions."
+sidebar_label: "Enhancement: dbt Cloud auto-installs 'dbt deps' on startup"
+tags: [Oct-2023, IDE]
+date: 2023-10-13
+sidebar_position: 06
+---
+
+The dbt Cloud IDE and dbt Cloud CLI now automatically installs `dbt deps` when your environment starts or when necessary. Previously, it would prompt you to run `dbt deps` during initialization. However, you still need to run `dbt deps` if you modify your `packages.yml` or `dependencies.yml` file.
```

Comment:
The "when necessary" here is a little vague, especially within the context of the next live they says they still need to run it manually in two scenarios that seem "necessary". Can we clarify that a little (or remove if it's not needed)?

Reply:
Thank you so much @matthewshaver ! I don't think it's a blocker for this release but would be good to get some context (I can't clearly remember the why it says 'when necessary') @greg-mckeon or @dave-connors-3  do you have addl info on what the 'when necessary' means for automatic dbt deps installation on the backend? I know sometimes dbt asks you to run deps so maybe it's those scenarios?